python_modules/dagster/dagster/_utils/yaml_utils.py

```diff
@@ -1,12 +1,19 @@
 import functools
 import glob
-from typing import Any, Dict, List, Mapping, Sequence, Type
+from typing import Any, Dict, List, Mapping, Sequence, Type, cast
 
 import yaml
 
 import dagster._check as check
 
 from .merger import deep_merge_dicts
+from .source_position import (
+    KeyPathSegment,
+    LineCol,
+    SourcePosition,
+    SourcePositionTreeNode,
+    ValueAndSourcePositionTreeNode,
+)
 
 YAML_TIMESTAMP_TAG = "tag:yaml.org,2002:timestamp"
```

Comment:
Might be worth noting why we strip this loader out somewhere

Reply:
👍